scene/gui/base_button.cpp

```diff
@@ -267,6 +261,10 @@ BaseButton::DrawMode BaseButton::get_draw_mode() const {
 		return DRAW_DISABLED;
 	}
 
+	if (in_shortcut_feedback) {
```

Comment:
This could be replaced by
```suggestion
	if (shortcut_feedback_timer && !shortcut_feedback_timer->is_stopped()) {
```
to avoid extra bool flag, but it's not important I guess.